Makefile

```diff
@@ -133,7 +133,7 @@ generate: controller-gen client-gen lister-gen informer-gen
 
 .PHONY: bin/kubectl-schemahero
 bin/kubectl-schemahero:
-	go build \
+	GOOS=linux GOARCH=amd64 go build \
```

Comment:
Remove the GOOS and GOARCH changes from this file to prevent breaking other architectures.

Reply:
removed the GOOS and GOARCH changes from the makefile @wolf31o2